minetest/minetest
# Cinematic camera keybinding ability in controls menu

- Goal of the PR
Make it easier for users to toggle cinematic camera.
- How does the PR work?
It adds the cinematic camera to the controls tab in the menu
- Does it resolve any reported issue?
No.
- If not a bug fix, why is this PR needed? What usecases does it solve?
It helps normal players to be able to toggle the cinematic camera easier.

Ready for Review.

## How to test

Compile. Set Cinematic camera to a keybinding of your choosing, and ensure it toggles. Properly.
